tzlocal/darwin.py

```diff
@@ -28,6 +28,13 @@ def __exit__(self, type, value, traceback):
 
 
 def _get_localzone(_root='/'):
+    tzenv = os.environ.get('TZ')
+    if tzenv:
+        try:
+            return pytz.timezone(tzenv)
```

Comment:
I think in that case we should support the same thing as we do for Linux, so we need to use the _tz_from_env() function from unix.py.

Reply:
Good idea, fixed :)